### ISSUE TITLE
Cypress Runner struck when I used chrome. Only the timer keeps running but test cases are not getting executed. 

### ISSUE THREAD

**Issue by tce-SalilDew** [cypress-io/cypress#23397]
### Current behavior

Cypress Runner struck when I used chrome. Only the timer keeps running but test cases are not getting executed. 

### Desired behavior

Test runner should run properly when I use chrome. On changing the browser it runs properly. 

Video link attached for the reference 
https://drive.google.com/file/d/1VoGyJS-FdWNtCxuIWTGeWJXMIu5yH3C_/view?usp=sharing

### Test code to reproduce

It is random. 

### Cypress Version

10.4.0

### Node version

14.15.3

### Operating System

11.6.7 Mac

### Debug Logs

```shell
https://drive.google.com/file/d/1VoGyJS-FdWNtCxuIWTGeWJXMIu5yH3C_/view?usp=sharing
```

**Comment by rachelruderman** [comment]
Hi @tce-SalilDew ! I'm sorry to hear you're having issues. Can you set up a reproducible example so I can take a deeper look on my end? https://github.com/cypress-io/cypress-test-tiny is helpful for that

**Comment by nagash77** [comment]
Unfortunately we have to close this issue due to inactivity. Please comment if there is new information to provide concerning the original issue and we can reopen.

**Comment by tce-SalilDew** [comment]
Hi @rachelruderman, 

Sorry for the late reply, Actually this issue is happening intermediately. Not sure about the actual cause / Steps. I was running my test cases one by one, By using "Only" and suddenly this happened. 

Actually when we were running our build on production using the chrome browser then we observe that the build was getting struck. Then we changed the browser to electron and from that day till now our build is running fine. In order to find the reason for the struck I decided to run my test cases one by one and on the cypress runner when I was running my test cases one by one using "only" and on the chrome browser, it gets struck. However, it does not happen every time for the same test case.
